makeconf/management/commands/makeconf.py

```diff
@@ -30,13 +32,25 @@ def _get_template_map(self):
             map_ = settings.MAKECONF_MAP
         except AttributeError:
             map_ = {}
+        try:
+            eb_modules = settings.MAKECONF_EB_MODULES
+            if os.path.isdir('.ebextensions'):
+                self.stdout.write('Removing directory .ebextensions')
+                shutil.rmtree('.ebextensions')
+            for count, module in enumerate(eb_modules):
+                map_[
```

Comment:
This line is complicated enough you may want to pull the dict key name out as a separate assignment?